src/Controller/OG.php

```diff
@@ -27,25 +27,31 @@ class OG {
    *   The entity type.
    * @param $bundle
    *   The bundle name.
+   * @param $settings
+   *   (Optional) allow overriding the default definition of the field/instance.
+   *   Allowed values:
+   *   - field: override the field definitions.
```

Comment:
Happens here - https://github.com/amitaibu/og/pull/20/files#diff-7f53500460992339c0cf8b640ee8b3faR40 and https://github.com/amitaibu/og/pull/20/files#diff-7f53500460992339c0cf8b640ee8b3faR58. I forgot to add in the other definitions.